console/console-init/ui/src/modules/iot-device-detail/components/ConnectionGateway/ConnectionGateway.tsx

```diff
@@ -17,20 +21,20 @@ import { OutlinedQuestionCircleIcon } from "@patternfly/react-icons";
 import { StyleSheet, css } from "aphrodite";
 export interface IConnectionGatewayProps {
   deviceList?: any;
-  deviceGroup?: any;
+  gatewayGroups?: any;
 }
 const styles = StyleSheet.create({
   card_body: {
-    marginBottom: 20
+    marginBottom: "20rem"
```

Comment:
You did wrong. You can't have the same number for rem. rem works based on root.
For example if you have root size 25px , in rem it will be 1rem

Please look  the changes in browser , it will cause the issue.